src/Bicep.LangServer.IntegrationTests/CodeActionTests.cs

```diff
@@ -325,22 +326,24 @@ public async Task VerifyCodeActionIsAvailableToSuppressCoreCompilerWarning()
 
         private static IEnumerable<TextSpan> GetOverlappingSpans(TextSpan span)
         {
+            // NOTE: These code assumes there are no errors in the code that are exactly adject to each other or that overlap
+
             // Same span.
             yield return span;
 
             // Adjacent spans before.
-            int startOffset = Math.Max(0, span.Position - 10);
-            yield return new TextSpan(startOffset, 10);
+            int startOffset = Math.Max(0, span.Position - 1);
```

Comment:
+/-10 before the spans is too much space - it can pick up other errors in the test code that are nearby, causing failures.  Reduced to +/-1.